projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -196,7 +196,9 @@ export function selectAccountDeliveryMode() {
 
   const orderReview = waitForPage('/checkout/review-order', 'getReviewOrder');
 
-  cy.get('.cx-checkout-btns button.btn-primary').should('be.enabled').click();
+  cy.get('.cx-checkout-btns button.btn-primary')
+    .should('be.enabled')
```

Comment:
I guess you checked this?

Reply:
yup same thing, but I kept I forgot if it was current or incoming, but it's the same. I kept the one with the force in the end (doesn't hurt)